scylla-cdc-base/src/main/java/com/scylladb/cdc/model/ExponentialRetryBackoffWithJitter.java

```diff
@@ -2,30 +2,33 @@
 
 import com.google.common.base.Preconditions;
 
-import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
 
 public class ExponentialRetryBackoffWithJitter implements RetryBackoff {
-    private final Random random;
     private final int maximumBackoffMs;
     private final int backoffBase;
+    private final int maxJitterPercentage;
```

Comment:
```suggestion
    private final int jitterPercentage;
```

Reply:
It also worth to limit jitter absolute value, say to 100ms

Reply:
Added maxJitterMs, but did not set any default value. 
Having max jitter of 100ms with backoff going up to 30000ms feels insignificant.
Every backoff value will be exactly (backoff - 100ms) after a few retries because 100 ms range is too small.